buildSrc/src/test/java/org/elasticsearch/gradle/test/BaseTestCase.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.elasticsearch.gradle.test;
+
+import com.carrotsearch.randomizedtesting.JUnit4MethodProvider;
+import com.carrotsearch.randomizedtesting.RandomizedRunner;
+import com.carrotsearch.randomizedtesting.annotations.TestMethodProviders;
+import org.junit.runner.RunWith;
+
+@RunWith(RandomizedRunner.class)
+@TestMethodProviders({
+    JUnit4MethodProvider.class,
+    JUnit3MethodProvider.class
+})
+public abstract class BaseTestCase {
```

Comment:
You can make this extend from `Assert` to not have to import the common methods used like assertEquals

Reply:
I'll fix these, thanks, I finally figured out  where these are coming from, it's in the template for test classes.